app/services/data_migrations/change_english_nationality_data.rb

```diff
@@ -0,0 +1,20 @@
+module DataMigrations
+  class ChangeEnglishNationalityData
+    TIMESTAMP = 20220525155807
+    MANUAL_RUN = false
+
+    def change
+      ApplicationForm.where(first_nationality: %w[English Welsh Scottish], recruitment_cycle_year: 2022).each do |nationality|
```

Comment:
How many records are we updating? `find_each` will retrieve in batches, consider using this if it's a huge resultset.